src/main/java/com/gregtechceu/gtceu/client/util/RenderUtil.java

```diff
@@ -118,4 +123,39 @@ public static Vector3f transformVertex(Vector3f vertex, Direction direction, flo
 
         return vec3f(vertex.x + addX, vertex.y + addY, vertex.z + addZ);
     }
+
+    public static Fluid fromRecipe(GTRecipe recipe) {
```

Comment:
I'd call this `getRenderableFluidFromRecipe`

Reply:
Intellij doesn't like the word renderable, how about `getRecipeFluidToRender`